docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md

```diff
@@ -0,0 +1,97 @@
+---
+id: inclusive-gateways
+title: "Inclusive gateway"
+description: "An inclusive gateway (or OR-gateway) allows you to make a decision based on data."
+---
+
+:::note
+
+Currently, we've add support for the diverging (i.e. splitting, forking) inclusive gateway, not yet add support for the converging (i.e. merging, joining) inclusive gateway.
+
+- In addition, a combination of parallel and exclusive gateways can be used to merge the flows again.
+
+:::
+
+The inclusive gateway (or OR-gateway) allows for making multiple decisions, not just one.
+
+![process](assets/inclusive-gateway.png)
+
+If an inclusive gateway has multiple outgoing sequence flows, all sequence flows must have a `conditionExpression` to define when the flow is taken. The gateway can have one sequence flow be defined as the default flow.
+
+When an inclusive gateway is entered, the `conditionExpression` is evaluated. The process instance takes all sequence flows where the condition is fulfilled.
+
+For example: Cources selected include `pasta` and `salad`
+
+![process](assets/inclusive-gateway-1.png)
+
+For example: Cources selected include `pasta`, `steak` and `salad`
+
+![process](assets/inclusive-gateway-3.png)
+
+For example: Cources selected include `steak` and `salad`
+
+![process](assets/inclusive-gateway-4.png)
+
+If no condition is fulfilled, it takes the **default flow** of the gateway. If the gateway has no default flow, an incident is created.
+
+![process](assets/inclusive-gateway-2.png)
+
+If the inclusive gateway only has one outgoing sequence flow, then it does not need to have a condition.
+
+## Conditions
+
+A `conditionExpression` defines when a flow is taken. It is a [boolean expression](/components/concepts/expressions.md#boolean-expressions) that can access the process instance variables and compare them with literals or other variables. The condition is fulfilled when the expression returns `true`.
+
+Multiple boolean values or comparisons can be combined as disjunction (`and`) or conjunction (`or`).
+
+For example:
+
+```feel
+= totalPrice > 100
+
+= order.customer = "Paul"
+
+= orderCount > 15 or totalPrice > 50
+
+= valid and orderCount > 0
+
+= conatins(courses ,"salad")
+```
+
+## Additional resources
+
+### XML representation
+
+An inclusive gateway with three outgoing sequence flows and the defulat sequence flow is `Salad`:
+
+```xml
+<bpmn:inclusiveGateway id="Gateway_1dj8ts6" name="Courses selected?" default="Flow_05d0jjq">
+      <bpmn:incoming>Flow_0mfam08</bpmn:incoming>
+      <bpmn:outgoing>Flow_0d3xogt</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1le3l31</bpmn:outgoing>
+      <bpmn:outgoing>Flow_05d0jjq</bpmn:outgoing>
+</bpmn:inclusiveGateway>
+<bpmn:sequenceFlow id="Flow_0d3xogt" name="Pasta"
+    sourceRef="Gateway_1dj8ts6" targetRef="Activity_1orhxob">
+    <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">
+       = conatins(courses ,"pasta")
```

Comment:
❌ Please fix the typo in `conatins` -> `contains` and reformat `(courses ,"pasta")` -> `(courses, "pasta")`